libs/designer/src/lib/core/actions/bjsworkflow/add.ts

```diff
@@ -332,11 +329,16 @@ const getOperationType = (operation: DiscoveryOperation<DiscoveryResultTypes>):
     : operationType;
 };
 
-export const getRootNodeDetails = (workflowState: WorkflowState, connectorId: string): string | undefined => {
-  if (workflowState.graph?.children && connectorId.indexOf(Constants.INVOKER_CONNECTION.DATAVERSE_CONNECTOR_ID) > -1) {
+export const getRootNodeManifest = async (
```

Comment:
rename to getTriggerNodeManifest, please do not use the term root here unless you want roots in child graphs